systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java

```diff
@@ -1948,6 +1948,94 @@ void testConsumerOffsetFiles() throws Exception {
         assertThat("Folder kafka-log0 doesn't contains 100 files", result, containsString(stringToMatch.toString()));
     }
 
+    @Test
+    void testLabelsAndAnnotationforPVC() {
+        final String labelAnnotationKey = "testKey";
+
+        Map<String, String> pvcLabel = new HashMap<>();
+        pvcLabel.put(labelAnnotationKey, "testValue");
+        Map<String, String> pvcAnnotation = pvcLabel;
```

Comment:
Using a separate map for each component (kafka, zoo) and each type (label, annotation) would help to test there are no typos such as using labels fields for annotations etc.